README.md

```diff
@@ -55,69 +48,37 @@ While RFCs are intended for "significant" changes, we recommend trying to keep e
-1. Once the value and trade-offs of the change have been sufficiently discussed, a non-author member of the relevant SIG will remove "WIP" from the PR's title and propose a "motion for final comment period" (FCP)
-    * By this point, no **new** points should have been raised in the discussion for **at least three business days**
-    * If substantial new points (e.g., a significant trade-off) are raised *during* the FCP, the FCP should be cancelled and discussion renewed
-1. During the FCP, SIG members should each vote to `approve`, `reject`, or `defer` the RFC
-    * If the discussion has been contentious, members should explain the rationale behind their vote
-1. The FCP will be considered closed and the RFC either approved, rejected, or approved after the following conditions have all been met:
-    * At least two business days have passed since the FCP began
-    * Either:
-      * A majority of non-author members have voted
-      * At least three non-author members have voted, and the vote is unanimous
-1. The RFC will be merged with the appropriate state
-
-### Reviewing an RFC
-
-Discussion should primarily take place in the pull request aimed at approving the RFC. Any discussion that takes place outside of GitHub - e.g., in meetings or community chat - should be summarized and added to the pull request.
-
-### Implementing an RFC
-
-Once an RFC has been approved, a corresponding issue should be created and prioritized accordingly in the relevant repository.
+### Writing an RFC
 
-The original author is welcome but not obligated to implement it. If you are interested in implementing an RFC, please mention it in either the RFC pull request or the new issue (depending on which is active by then). While authors who are interested in implementation generally get priority, we encourage anyone who's interested to collaborate together!
+* First, [fork](https://help.github.com/en/articles/fork-a-repo) this [repo](https://github.com/open-telemetry/oteps).
+* Copy [`0000-template.md`](./0000-template.md) to `text/0000-my-rfc.md`, where `my-rfc` is a title relevant to your proposal, and `0000` is the RFC ID. Leave the number as is for now. Once a Pull Request is made, update this ID to match the PR ID.
+* Fill in the template. Put care into the details: It is important to present convincing motivation, demonstrate an understanding of the design's impact, and honestly assess the drawbacks and potential alternatives.
 
-If neither an author nor another community member asks to implement the RFC, it will be assigned once it gets high enough in the backlog.
+### Submitting the RFC
+* An RFC is `proposed` by posting it as a PR. Once the PR is created, update the RFC file name to use the PR ID as the RFC ID.
+* An RFC is `approved` when four reviewers github-approve the PR. The RFC is then merged.
```

Comment:
Is requiring four approvers new in this PR?

Reply:
No, that is what we currently have.